app/routers/hello.py

```diff
@@ -21,3 +21,15 @@ def initdb():
 
     create_db_and_tables()
     return {"message": "Database and tables created"}
+
+
+@router.get(
```

Comment:
This is helpful, however, I'd change it to POST method just bc we're not returning anything
```suggestion
@router.post(
```